roboconf-dist-rpm-agent/src/rpm/control/postinst

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+#
+#  Copyright 2014 Linagora, Université Joseph Fourier
```

Comment:
Ok. May be replace also Université Joseph Fourier by Université Grenoble Aples ? Indeed, since January 2016 UJF changed name and became UGA.